templates/workloads/services/worker/manifest.yml

```diff
@@ -0,0 +1,60 @@
+# The manifest for the "{{.Name}}" service.
+# Read the full specification for the "Worker Service" type at:
+# https://aws.github.io/copilot-cli/docs/manifest/worker-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: {{.Name}}
+type: {{.Type}}
+
+# Configuration for your containers and service.
+image:
+{{- if .ImageConfig.Build.BuildArgs.Dockerfile}}
+  # Docker build arguments.
+  build: {{.ImageConfig.Build.BuildArgs.Dockerfile}}
+{{- end}}
+{{- if .ImageConfig.Image.Location}}
+  location: {{.ImageConfig.Image.Location}}
+{{- end}}
+{{- if .ImageConfig.HealthCheck}}
+  healthcheck:
+    # Container health checks
+    command: {{fmtSlice (quoteSlice .ImageConfig.HealthCheck.Command)}}
+    interval: {{.ImageConfig.HealthCheck.Interval}}
+    retries: {{.ImageConfig.HealthCheck.Retries}}
+    timeout: {{.ImageConfig.HealthCheck.Timeout}}
+    start_period: {{.ImageConfig.HealthCheck.StartPeriod}}
+{{- end}}
+
+cpu: {{.CPU}}       # Number of CPU units for the task.
+memory: {{.Memory}}    # Amount of memory in MiB used by the task.
+count: {{.Count.Value}}       # Number of tasks that should be running in your service.
+exec: true     # Enable running commands in your container.
+
+{{- if .Subscribe}}{{- if .Subscribe.Topics}}
+subscribe:
+  topics:
+{{- range $topic := .Subscribe.Topics}}
+    - name: {{.Name}}
+      service: {{.Service}}
```

Comment:
This might not work....

Reply:
Would it need an additional space like the [pipelines](https://aws.github.io/copilot-cli/docs/manifest/pipeline/)?

Reply:
My plan is that the user will be able to pick from any available topics in init, so those can be used to fill the manifest after init here.

Reply:
Yeah but should we use `$topic`? Otherwise `.Name` might just be the service name.

Reply:
Good point - I can toss this adjustment to $topic.Name into the worker service PR up now.